src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,15 @@
+@keyframes minutes {
+  100% {
```

Comment:
Please and 0% point for animation, because arrows very slow down in the interval of 50-60 seconds